apps/etterlatte-statistikk/src/main/kotlin/statistikk/config/ApplicationContext.kt

```diff
@@ -43,19 +45,46 @@ class ApplicationContext {
     }
 
     private val behandlingClient: BehandlingClient by lazy {
-        BehandlingClientImpl(httpClient)
+        BehandlingClientImpl(behandlingHttpClient)
     }
 
-    private val statistikkRepository: StatistikkRepository by lazy {
-        StatistikkRepository(datasource)
+    private val statistikkRepository: StoenadRepository by lazy {
+        StoenadRepository(datasource)
     }
 
-    private val sakstatistikkRepository: SakstatistikkRepository by lazy {
-        SakstatistikkRepository(datasource)
+    private val sakstatistikkRepository: SakRepository by lazy {
+        SakRepository(datasource)
     }
     private val datasource: DataSource by lazy { DataSourceBuilder.createDataSource(env).also { it.migrate() } }
 
-    private val httpClient: HttpClient by lazy {
+    private val beregningClient: BeregningClient by lazy {
+        BeregningClientImpl(beregningHttpClient)
+    }
+
+    private val beregningHttpClient: HttpClient by lazy {
+        HttpClient(OkHttp) {
+            expectSuccess = true
+            install(ContentNegotiation) {
+                jackson {
+                    registerModule(JavaTimeModule())
+                    disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+                    disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
+                }
+            }
+            install(Auth) {
+                clientCredential {
+                    config = env.toMutableMap()
+                        .apply { put("AZURE_APP_OUTBOUND_SCOPE", requireNotNull(get("BEREGNING_AZURE_SCOPE"))) }
+                }
+            }
+            defaultRequest {
+                header(X_CORRELATION_ID, getCorrelationId())
+                url("http://etterlatte-beregning/")
```

Comment:
No biggie, men denne default-urlen er vel strengt talt ikke nødvendig

Reply:
Er den ikke? Synes å huske å få feil uten denne i kommunikasjon mellom apper med grunnlagKlient i behandling

Reply:
Går uansett heller over til httpClientClientCredentials for denne